homeassistant/components/binary_sensor/zwave.py

```diff
@@ -63,6 +63,11 @@ def should_poll(self):
         """No polling needed."""
         return False
 
+    @property
```

Comment:
Would it make sense to add this to the Z-Wave entity base class instead of adding it to each derived entity?